raytrace/src/main/scala/org/scalajs/benchmark/raytrace/RayTrace.scala

```diff
@@ -14,9 +14,20 @@
 // Ported from the v8 benchmark suite by Google 2012.
 // Ported from the Dart benchmark_harness to Scala.js by Jonas Fonseca 2013
 
-package org.scalajs.benchmark.tracer
+package org.scalajs.benchmark.raytrace
 
-object Tracer extends org.scalajs.benchmark.Benchmark with App {
+import org.scalajs.benchmark._
+import scalajs.js
+import scalajs.js.annotation.JSExport
+
+object RayTrace extends App with js.JSApp {
```

Comment:
No, it is just badly named, It is a trait that contains some binding for `raytrace/index.html` and `raytrace/index-fastopt.html`.